charts/capsule-proxy/templates/certgen-job.yaml

```diff
@@ -5,8 +5,14 @@ metadata:
   name: {{ include "capsule-proxy.fullname" . }}-certgen
   labels:
     {{- include "capsule-proxy.labels" . | nindent 4 }}
+  {{- with .Values.jobs.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
 spec:
+  {{- if .Values.jobs.deleteAfterFinished }}
   ttlSecondsAfterFinished: 60
+  {{- end }}
```

Comment:
Wouldn't be better to have this value configurable? Something has:

```
{{- if gt .Values.jobs.ttlSecondsAfterFinished 0 }}
  ttlSecondsAfterFinished: {{ .Values.jobs.ttlSecondsAfterFinished }}
{{- end }}
```

Reply:
Can do this, yes =)

Reply:
@prometherion Can you take another look? Because `0` is also valid (delete immediately) I used `ge` instead of `gt`.  
For disabling I used `-1`.

Reply:
Definitely makes sense, you're right, and it's already documented.